package.json

```diff
@@ -37,7 +37,7 @@
     "jscoverage": "^0.6.0",
     "mocha": "^2.3.4",
     "mocha-lcov-reporter": "^1.0.0",
-    "protagonist": "",
```

Comment:
Example, when we drop the blueprint AST we will be releasing `2.0.0` and your tests are using that AST.